src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -57,6 +61,14 @@ class PipelineInfo:
     failures: str
 
 
+@dataclass
```

Comment:
Do we want to capture only the failing ones or all of them with the failing one anotated?